docs/en/client/other-language-client-user-guide.md

```diff
@@ -6,7 +6,7 @@ Also, if any team/individual is interested, they are welcome to help us to imple
 
 ## 1.1 Application access to Apollo
```

Comment:
There are also links to `en/usage/other-language-client-user-guide` in this document as well as in `en/design/apollo-core-concept-namespace.md` and `en/portal/apollo-user-guide.md` that needs to be replaced.